spec/models/user_spec.rb

```diff
@@ -379,31 +379,35 @@ def user_from_authorization_service(service_name, signed_in_resource, cta_varian
       expect(new_user.identities.size).to eq(2)
     end
 
-    context "with active jobs" do
-      before { ActiveJob::Base.queue_adapter = :inline }
-
+    context "when estimating the default language" do
       it "estimates default language to be nil" do
-        user.estimate_default_language!
-        expect(user.estimated_default_language).to eq(nil)
+        perform_enqueued_jobs do
+          user.estimate_default_language!
+        end
+        expect(user.reload.estimated_default_language).to eq(nil)
       end
 
       it "estimates default language to be japan with jp email" do
-        user.update_column(:email, "ben@hello.jp")
-        user.estimate_default_language!
-        user.reload
-        expect(user.estimated_default_language).to eq("ja")
+        perform_enqueued_jobs do
+          user.update_column(:email, "ben@hello.jp")
+          user.estimate_default_language!
+        end
+        expect(user.reload.estimated_default_language).to eq("ja")
       end
 
       it "estimates default language based on ID dump" do
-        new_user = user_from_authorization_service(:twitter, nil, "navbar_basic")
-        new_user.estimate_default_language!
+        perform_enqueued_jobs do
+          new_user = user_from_authorization_service(:twitter, nil, "navbar_basic")
+          new_user.estimate_default_language!
+        end
```

Comment:
This test doesn't have an expect clause ? 🤔

Reply:
You're right, It didn't have it before either, I thought it was one of those "testing that this does not raise an error" tests so I let it be. I'm not familiar with the code so I don't know what's that supposed to expect

Reply:
I'm currently working on estimating languages #2369 , so I can add an expectation for this spec in my branch.